spec/schemas/_common.query_dsl.yaml

```diff
@@ -1237,7 +1237,6 @@ components:
           type: string
         query_image:
           type: string
-          format: binary
```

Comment:
This should be replaced by `contentEncoding: base64` (https://swagger.io/specification/#:~:text=In%20contrast%20with%20the%203.0,in%20the%20Content%2DType%20header.)

Reply:
Adding this.

Reply:
I thought it was related to the content encoding of the entire body... but we can easily verify this once @urinud added an end-to-end test by putting non-base64-encoded data into the test and seeing a schema validation failure.